src/Calculator.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react'; import Button from './components/Button';
+import Screen from './components/Screen';
+import Header from './components/Header';
+import calculate from './logic/calculate';
+
+const Calculator = () => {
+  const [total, setTotal] = useState(null);
+  const [next, setNext] = useState(null);
+  const [operation, setOperation] = useState(null);
+
+  const getState = () => (
+    { total, next, operation }
+  );
+
+  const buttonClick = (value) => {
+    const oper = calculate(getState(), value);
+    const { total: sum, next: nextVal, operation: validOperation } = oper;
+    if (sum !== undefined) setTotal(sum);
+    if (nextVal !== undefined) setNext(nextVal);
+    if (validOperation !== undefined) setOperation(validOperation);
+  };
+
```

Comment:
- [ ] Kindly make this component a class-based component as required by the project requirements. For this project, you are required to make use of React class components for all your components.
- [ ] Also, I noticed the `Calculator.js` component is outside of the components directory. Kindly move it to the components directory and then import it from that directory into the AApp.js where you making use of it. 👍 

**Please Read the Screenshot From Your Project Requirements below**

![image](https://user-images.githubusercontent.com/50941074/171285327-d0022382-d9dd-4ad8-a2bb-51154c49bc71.png)